websites/J/JKAnime/iframe.ts

```diff
@@ -0,0 +1,16 @@
+const iframe = new iFrame();
+
+iframe.on("UpdateData", async () => {

```

Comment:
Codacy found an issue: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/gh/PreMiD/Presences/pullRequest?prid=6113512)